spec/support/features/pages/finance/finance_payment_breakdown_report.rb

```diff
@@ -51,7 +53,7 @@ def has_retained_1_declaration_payment_table?(num_ects: 0, num_mentors: 0, num_d
     end
 
     def has_other_fees_table?(num_ects: 0, num_mentors: 0)
-      adjustments_panel.has_uplift_payments? num_ects + num_mentors
```

Comment:
could we add a step for the adjustments panel now we switched this up to uplifts?